index.d.ts

```diff
@@ -1579,6 +1582,277 @@ declare namespace plugins {
   interface winston extends Integration {}
 }
 
+export namespace opentelemetry {
```

Comment:
There's probably a way we could just link to OTel's API docs instead of including it directly here, but this should suffice for now.